python/mdvtools/auth/authutils.py

```
@@ -0,0 +1,359 @@
+                if error_response:
+                    logger.error(error_response)
+                    return error_response
+            elif auth_method == "shibboleth":
+                user, error_response = validate_sso_user(request)
+                if error_response:
+                    logger.error(error_response)
+                    return error_response
+            else:
+                return jsonify({"error": "Unknown authentication method"}), 400
+
+            return f(user=user, *args, **kwargs)
```

Ruff (0.8.2)

Star-arg unpacking after a keyword argument is strongly discouraged

(B026)